File: 12.py
#Павлов Арсений, задача 12
t1 = float(input('широта первой точки'))
g1 = float(input('долгота первой точки'))
t2 = float(input('широта второй точки'))
g2 = float(input('долгота второй точки'))
from math import pi, acos, sin, cos
t1 = t1 * pi / 180
g1 = t1 * pi / 180
t2 = t1 * pi / 180
g2 = t1 * pi / 180
print('расстояние между этими точками равно', 6371.01 * acos(sin(t1) * sin(t2) + cos(t1) * cos(t2) * cos(g1 - g2)))

File: 12.cpp
// самраб 1 прод 12
```

#include <iostream>

#include <cmath>

using namespace std;

int main()

{

double m, a, u, F0, Ft, F, a_new;

cout << "Введите массу тела m: ";

cin >> m;

cout << "Введите угол a (в градусах): ";

cin >> a;

a = a * M_PI / 180.0; // переводим градусы в радианы

cout << "Введите коэффициент трения u: ";

cin >> u;

cout << "Введите силу F0: ";

cin >> F0;

Ft = u * m * 9.81 * sin(a);

F = F0 * cos(a) - Ft;

a_new = (abs(F) < 0.0001) ? 0.0 : F / m;

cout << "Ускорение тела: " << a_new << " м/c²" << endl;

return 0;

}